CODE/Logicytics.py

```
@@ -286,69 +288,93 @@
     else:
         try:
-            execution_list = generate_execution_list(action)
-            for file in range(len(execution_list)):  # Loop through List
-                log.parse_execution(Execute.script(execution_list[file]))
-                log.info(f"{execution_list[file]} executed")
+            EXECUTION_LIST = generate_execution_list()
+            for file in range(len(EXECUTION_LIST)):  # Loop through List
+                log.parse_execution(Execute.script(EXECUTION_LIST[file]))
+                log.info(f"{EXECUTION_LIST[file]} executed")
         except UnicodeDecodeError as e:
             log.error(f"Error in code: {e}")
```

Ruff (0.8.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)